contracts/hackatom/src/contract.rs

```diff
@@ -84,9 +84,8 @@ pub fn init<S: Storage, A: Api, Q: Querier>(
 
     // This adds some unrelated data and log for testing purposes
     Ok(InitResponse {
-        data: Some(vec![0xF0, 0x0B, 0xAA].into()),
         log: vec![log("Let the", "hacking begin")],
-        ..InitResponse::default()
+        messages: vec![],
```

Comment:
Can you move this test `data` to one of the handle implementations (probably `do_release`)? Then we can test this ends up in the REST response data